commands/initial/index.js

```diff
@@ -19,19 +19,19 @@ module.exports = (rootDir) => {
             { name: 'examples/index.html' },
             { blank: true, name: 'examples/styles.scss' },
             { name: 'index.ts' },
-            { name: 'karma.conf.js' },
+            { name: 'karma.conf.js', overwrite: true },
             { destination: path.resolve(srcDir, '{{ name }}.module.ts'), name: 'src/module.ts' },
-            { name: 'package.json' },
+            { name: 'package.json', update: true },
```

Comment:
erector-set is not going to update the file with this representation...docs haven't been updated yet...

but it accepts one of two value types. you can provide a known updater as a string or a custom function. right now the only updater is 'json'. The [json updater](https://github.com/gonzofish/erector-set/blob/master/src/updaters/json.js) will just merge the JSON objects.

Look at the [erector-set example for a custom updater](https://github.com/gonzofish/erector-set/blob/master/examples/build/index.js#L9). This seems more like what we need to do, since we need to ensure we keep to dependencies, scripts, etc. of the template but keep the user-added scripts, dependencies, etc.